src/pages/home.js

```diff
@@ -1,24 +1,28 @@
-function Home() {
-  const CURRENT_USER_URL = "http://localhost:3000/users/1";
-
-  async function getAPIData() {
-    const response = await fetch(CURRENT_USER_URL, {
-      credentials: "same-origin",
-      headers: { Authorization: `Bearer ${localStorage.getItem("token")}` },
-    });
-    const json = await response.json();
-    return json;
-  }
+import "./font.css";
+import NavBar from "../components/navBar";
+import { useNavigate } from "react-router-dom";
 
-  const user = async () => {
-    console.log(await getAPIData().then((response) => response["id"]));
-  };
-
-  user();
+function Home() {
+  let navigate = useNavigate();
 
   return (
-    <div className="bg-cod-gray w-full h-full">
-      <h1 className="text-center text-white"> Weekly Task</h1>
+    <div className="bg-cod-gray font absolute h-full w-full">
+      <NavBar />
+      <h1 className="text-white m-32 text-5xl">Space Center Activities</h1>
+      <div className="m-32 flex gap-32">
+        <button
```

Comment:
Don't navigate using `button`. It's best practice to use regular `<a>` tags, which are meant for actual navigation. In this case, as it's a react app, use the  provided `Link` element with a normal href.

Reply:
The `navigate` function is still useful in some cases, like navigating after running some logic in a function, but it's not meant for normal UI navigation